Bottom: 621a38603da5cef2371ba5e64b81ac804885f8e2
Top:    9303880701bae1cd8bf23f0db31fbcde4db1d4b1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-10 16:02:03 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 795b9fe..c4b314e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
